.github/workflows/main.yaml

```
@@ -0,0 +1,20 @@
+name: "Continuous Build"
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Wait, shouldn't we use concrete versions?
```suggestion
    runs-on: ubuntu-20.04
```

---

hm. we're using latest for our pr builds....this was just a copy of that. I'll update both.